src/app/pages/task-calendar/initshutdown/initshutdown-form/initshutdown-form.component.ts

```diff
@@ -79,7 +79,7 @@ export class InitshutdownFormComponent {
     type: 'checkbox',
     name: 'ini_enabled',
     placeholder: T('Enabled'),
-    tooltip: T('Unset to diable the task without deleting it.'),
+    tooltip: T('Unset to disable the task without deleting it.'),
```

Comment:
This is very confusing.  The field is called enable, but is described with negative logic.  Flip it around.

Enable this job. Unset to prevent the job from running.

Reply:
done